AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -41,10 +47,12 @@ - (instancetype)init
 
     _titleNode = [[ASTextNode alloc] init];
     _imageNode = [[ASImageNode alloc] init];
+    _backgroundImageNode = [[ASImageNode alloc] init];
```

Comment:
@1nput0utput  Just realized this - but indeed we should be able to set .isLayerBacked = YES on all of these!  Because the ASButtonNode itself is an ASControlNode, it will do all of the touch handling that we need in this class.  Doing this would significantly reduce the overhead of this simpler design of creating all three of these nodes up front even if a given use case only requires one of them.